bottlerocket-settings-models/modeled-types/src/kubernetes.rs

```diff
@@ -1581,7 +1581,7 @@ pub struct NvidiaGpuModel {
 }
 
 lazy_static! {
-    pub(crate) static ref NVIDIAGPU_NAME: Regex = Regex::new(r"^([a-z])(\d+)\.(\d+)gb$").unwrap();
+    pub(crate) static ref NVIDIAGPU_NAME: Regex = Regex::new(r"^([a-z]+)(\d+)\.(\d+)gb$").unwrap();
```

Comment:
```suggestion
    pub(crate) static ref NVIDIAGPU_NAME: Regex = Regex::new(r"^([a-z0-9]+)\.(\d+)gb$").unwrap();
```
Would it be safer to assume any combination of letters and numbers before the dot is valid? It looks like NVIDIA had a **a100x** at one point.

A stricter version could be `^([a-z]+)(\d+)0([a-z])?\.(\d+)gb$`

Reply:
I think we can relax this more. I kept it to the known change I am aware of but we could relax this more to just alpha-numeric before the dot. Ultimately the migmanager will need to validate this.